apps/mark-scan/backend/schema.sql

```diff
@@ -39,3 +39,7 @@ create table ui_string_audio_ids (
   data text not null, -- JSON blob - see libs/types/UiStringAudioIdsSchema
   foreign key (language_code) references languages(code)
 );
+
+create table hardware_config (
+  is_pat_device_connected integer
+);
```

Comment:
This state was previously stored in the state machine `context`, but since we now need it on the frontend this felt like an appropriate place to put it.

Reply:
That's reasonable. I think it's unintuitive to the reader if that state is stored in memory in the state machine context, but I think it makes sense as a class property of `Store`.